upf/problem.py

```diff
@@ -135,6 +135,13 @@ def add_action(self, action: upf.Action):
                 self._kind.set_conditions_kind('NEGATIVE_CONDITIONS') # type: ignore
             if op.OR in ops:
                 self._kind.set_conditions_kind('DISJUNCTIVE_CONDITIONS') # type: ignore
+        for e in action.effects():
```

Comment:
You also need to update the taxonomy wiki: https://github.com/aiplan4eu/upf/blob/master/docs/wiki/Planning-Taxonomy.md

Reply:
I opened another pull-request on the docs/wiki repository